requirements.txt

```diff
@@ -1 +1,2 @@
 ops >= 1.5.0
+pydantic
```

Comment:
Should we update the version pin for `ops` to be `2` and add at least a major version pin for `pydantic`?

Reply:
Updated, thanks!